app/routes/git_tokens.py

```
@@ -4,340 +4,354 @@
+			id=git_label_uuid, user_id=authenticated_user.id
+		).first()
+		if git_label:
+			await git_label.delete()
+		else:
+			return APIResponse.error(
+				message=constants.TOKEN_NOT_FOUND,
+				status_code=status.HTTP_404_NOT_FOUND,
+			)
+		
+		return APIResponse.success(message=constants.TOKEN_DELETED_SUCCESSFULLY)
+	except ValueError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)